src/main/java/com/homework6/homework6/HelloworldController.java

```diff
@@ -3,10 +3,23 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.time.LocalDateTime;
+import java.time.ZonedDateTime;
+
 @RestController
 public class HelloworldController {
-    @GetMapping("/hello")
-    public String hello(){
-        return "Hello World";
+    @GetMapping("/local")
+
+    public String local(){

```

Comment:
意味のかたまりを意識して改行するようにしましょう 🚀 
```suggestion

    @GetMapping("/local")
    public String local(){

```